rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-pipeline-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+    - name: pipeline_alerts
+      interval: 1m
+      rules:
+        - alert: HighSchedulingOverhead
+          expr: |
+            (sum(sum_over_time(pipelinerun_duration_scheduled_seconds_sum{status='succeded'}[30m]))
+            /
+            sum(sum_over_time(pipelinerun_duration_scheduled_seconds_count{status='succeded'}[30m])))
+            /
+            (sum(sum_over_time(tekton_pipelines_controller_pipelinerun_duration_seconds_sum{status='success'}[30m]))
+            /
+            sum(sum_over_time(tekton_pipelines_controller_pipelinerun_duration_seconds_count{status='success'}[30m])))  > 0.05
+          for: 12h
+          labels:
+            severity: warning
+            slo: true
+          annotations:
+            summary: >-
+              Tekton has a scheduling overhead vs. PipelineRun durations of >5%
+            description: >-
+              Tekton controller is requiring more
+              than 5% of PipelineRun execution time to receive creation events.
+            runbook_url: TBD
+        - alert: HighExecutionOverhead
+          expr: |
+            (sum(sum_over_time(pipelinerun_gap_between_taskruns_milliseconds_sum{status='succeded'}[30m])/1000)
+            /
+            sum(sum_over_time(pipelinerun_gap_between_taskruns_milliseconds_count{status='succeded'}[30m])))
+            /
+            (sum(sum_over_time(tekton_pipelines_controller_pipelinerun_duration_seconds_sum{status='success'}[30m]))
+            /
+            sum(sum_over_time(tekton_pipelines_controller_pipelinerun_duration_seconds_count{status='success'}[30m])))  > 0.05
+          for: 12h
+          labels:
+            severity: warning
+            slo: true
+          annotations:
+            summary: >-
+              Tekton has a execution overhead vs. PipelineRun durations of >5%
```

Comment:
```suggestion
              Tekton has an execution overhead vs. PipelineRun durations of >5%
```

Reply:
You can use `{{ $value }}` to also show the calculated percentage.

Reply:
ok, and put that in the description, correct?   I'll add that in my next push to the branch/PR thanks

Reply:
indeed